include/seqan3/range/view/convert.hpp

```diff
@@ -64,37 +64,26 @@ namespace seqan3::view
  * Does not modify data.
  * \par Example
  * ```cpp
- * dna4_vector vec = "ACTTTGATA"_dna4;
- * auto v = vec | view::convert<char>;
- * std::cout << v << '\n'; // [A,C,T,T,T,G,A,T,A]
- * auto v2 = vec | view::convert<underlying_rank_t<dna4>> | view::convert<unsigned>;
- * std::cout << v2 << '\n'; // [0,1,3,3,3,2,0,3,0]
+ *   std::vector<int>  vec{7, 5, 0, 5, 0, 0, 4, 8, -3};
  *
- * std::vector<illumina18> qvec{{0}, {7}, {5}, {3}, {7}, {4}, {30}, {16}, {23}};
- * auto v3 = qvec | view::convert<char>;
- * std::cout << v3 << '\n'; // [!,(,&,$,(,%,?,1,8]
+ *   // pipe notation
+ *   auto v = vec | view::convert<bool>; // == [1, 1, 0, 1, 0, 0, 1, 1, 1];
  *
- * std::vector<dna4q> qcvec{{dna4::C, 0}, {dna4::A, 7}, {dna4::G, 5}, {dna4::T, 3}, {dna4::G, 7}, {dna4::A, 4}, {dna4::C, 30}, {dna4::T, 16}, {dna4::A, 23}};
- * auto v4 = qcvec | view::convert<char>;
- * std::cout << v4 << '\n'; // [C,A,G,T,G,A,C,T,A]
- * auto v5 = qcvec | view::convert<underlying_rank_t<dna4q>> | view::convert<unsigned>;
- * std::cout << v5 << '\n'; // [1,28,22,15,30,16,121,67,92]
- * auto v6 = qcvec | view::convert<dna4> | view::convert<char>;
- * std::cout << v6 << '\n'; // [C,A,G,T,G,A,C,T,A]
- * auto v7 = qcvec | view::convert<dna4> | view::convert<underlying_rank_t<dna4>> | view::convert<unsigned>;
- * std::cout << v7 << '\n'; // [1,0,2,3,2,0,1,3,0]
- * auto v8 = qcvec | view::convert<illumina18> | view::convert<char>;
- * std::cout << v8 << '\n'; // [!,(,&,$,(,%,?,1,8]
- * auto v9 = qcvec | view::convert<illumina18> | view::convert<underlying_rank_t<illumina18>> | view::convert<unsigned>;
- * std::cout << v9 << '\n'; // [0,7,5,3,7,4,30,16,23]
+ *   // function notation and immediate conversion to vector again
+ *   std::vector<bool> v2(view::convert<bool>(vec));
+ *
+ *   // combinability
+ *   std::vector<bool> cmp2{1, 1, 1, 0, 0, 1, 0, 1, 1};
+ *   auto v3 = vec | view::convert<bool> | ranges::view::reverse; // == [1, 1, 1, 0, 0, 1, 0, 1, 1];
```

Comment:
Because now it just performs casts and not conversion to/from rank/char types.

I could re-introduce examples with cast from dna4q to dna4, or between nucleotides, but I am not sure it adds readability... what do you think?